doxygen/contributor_help_pages/getting_started.md

```diff
@@ -250,15 +281,15 @@ There's a lot going on here particular to Stan math and so we will talk about ea
 
 ### (1) Arena memory
```

Comment:
I'd like to leave these, they are made so that each number corresponds with a Stan Math specific thing we want to discuss from the function written above it.